astropy/coordinates/tests/test_polarization.py

```diff
@@ -36,7 +36,7 @@ def test_vector_list_init():
 
 def test_undefined():
     sk = StokesCoord(np.arange(-10, 7))
-    assert_equal(sk.symbol, "? ? YX XY YY XX LR RL LL RR ? I Q U V ? ?".split())
+    assert_equal(sk.symbol, "? ? YX XY YY XX LR RL LL RR ? I Q U V ? ?".split())  # noqa: SIM905
```

Comment:
Instead of the ignore, is there a way to address it?

Reply:
Yes, it was discussed at lenght in https://github.com/astropy/astropy/pull/17891, but a noqa was the preffered solution.